moped-editor/src/utils/socrataHelpers.js

```diff
@@ -29,3 +29,33 @@ export const useSocrataGeojson = url => {
   error && console.error(error);
   return { features, error, loading: !features && !error };
 };
+
+/**
+ * Fetches data from a socrata json endpoint
+ * @param {String} url - The url endpoint
+ * @return {Object} data (json object), loading, and error states
+ */
+export const useSocrataJson = url => {
```

Comment:
@mateoclarke this looks identical to `useSocrataGeojson`, right? I think we can nix `useSocrataGeojson` and use this everywhere, you'll just need to swap out a few things for the signal autocomplete stuff.

Reply:
will do.

Reply:
I was a little stumped bc they appear identical but replacing them breaks the signal autocomplete. I think there's a smart way to abstract and make this dual purpose, but the way the geojson expects a feature attribute of the result is significant in this case. I could do some more nested if/else but I think I'll leave it for now.

![Screen Shot 2022-01-11 at 1 55 04 AM](https://user-images.githubusercontent.com/5697474/148903320-2c15900c-0dbb-4443-9b12-31e6674e0381.png)

Reply:
ah, yes. i think the format of the response JSON is merely dependent on the file extension you put in the endpoint url (`.json` or `.geojson`). i'll open an issue.